lvsfunc/deblock.py

```diff
@@ -182,7 +192,8 @@ def vsdpir(clip: vs.VideoNode, strength: VSDPIR_STRENGTH_TYPE = 25, mode: str =
     :return:                Deblocked or denoised clip in either the given clip's format or YUV444PS
     """
     try:
-        from vsmlrt import DPIR, Backend, DPIRModel
+        from vsmlrt import (Backend, DPIRModel, calc_tilesize, inference,
```

Comment:
Maybe also worth it to look at supporting not-vsmlrt packages? I recall people complaining that they can't use vsdpir anymore because vsmlrt doesn't work for them.

Reply:
I don't know if you've been following, but [holywu](https://github.com/HolyWu/vs-dpir) switched to the ONNX (CUDA, TensorRT, DirectML) runtime, just in python instead of a plugin.
It however supports AMD MiGraphX, so that might be cool to add.

Reply:
I haven't actually, but yeah, that'd be a cool additional support and give users the ability to fall back to something else that works if vsmlrt does not. If you want to implement it, you have my full support!

Reply:
Moved to #133